src/nvim/decoration.c

```diff
@@ -826,7 +826,7 @@ static void buf_signcols_validate_range(buf_T *buf, int row1, int row2, int add)
   int *overlap = xcalloc(sizeof(int), (size_t)(row2 + 1 - row1));
 
   // First find the number of overlapping signs at "row1".
-  (void)marktree_itr_get_overlap(buf->b_marktree, currow, 0, itr);
+  marktree_itr_get_overlap(buf->b_marktree, currow, 0, itr);
```

Comment:
Removing this (and others) may trigger a coverity warning. This was [added](https://github.com/neovim/neovim/pull/26518) recently to suppress one at least.

Reply:
We should disable this dumb warning then IMO rather than try to appease coverity.